plugins/UsersManager/Model.php

```diff
@@ -43,7 +52,8 @@ class Model
     public function __construct()
     {
         $this->passwordHelper = new Password();
-        $this->table = Common::prefixTable(self::$rawPrefix);
+        $this->userTable = Common::prefixTable(self::$rawPrefix);
+        $this->tokenTable = Common::prefixTable('user_token_auth');
```

Comment:
@diosmosis I think we have this in various places like this and it was already there before for `$this->table` so it might not be an issue in this case?

Reply:
It's not a big deal, can always be changed if it becomes a problem.